Wallapop Architecture/Scene.xctemplate/___VARIABLE_sceneName___Presenter.swift

```diff
@@ -1,25 +1,27 @@
 import Foundation
 import RxSwift
-import Common
 import CommonUI
 
 protocol ___VARIABLE_sceneName___Presenting: Presenting {
-    /* weak */ var ui: ___VARIABLE_sceneName___UI? { get set }
+    var ui: ___VARIABLE_sceneName___UI? { get set }
 }
 
-protocol ___VARIABLE_sceneName___UI: UI {
+protocol ___VARIABLE_sceneName___UI: UI, FeedbackShowable {
+    func apply(_ state: ___VARIABLE_sceneName___ViewModel)
```

Comment:
Same comment from filip applies here
```suggestion
    func display(_ viewModel: ___VARIABLE_sceneName___ViewModel)
```